Sources/Hummingbird/Environment.swift

```diff
@@ -111,22 +112,8 @@ public struct Environment: Sendable, Decodable, ExpressibleByDictionaryLiteral {
     /// Construct environment variable map
     static func getEnvironment() -> [String: String] {
         var values: [String: String] = [:]
-        let equalSign = Character("=")
-        #if canImport(Musl)
-        guard let envp = environ else { return [:] }
-        #else
-        let envp = environ
-        #endif
-        var idx = 0
-
-        while let entry = envp.advanced(by: idx).pointee {
-            let entry = String(cString: entry)
-            if let i = entry.firstIndex(of: equalSign) {
-                let key = String(entry.prefix(upTo: i))
-                let value = String(entry.suffix(from: i).dropFirst())
-                values[key.lowercased()] = value
-            }
-            idx += 1
+        for key in ProcessInfo.processInfo.environment.keys {
```

Comment:
You could do this as 
```swift
for item in ProcessInfo.processing.environment {
```
And then use `item.key` and `item.value` to avoid the additional dictionary look up.

Reply:
Sure, good call.